autopush-common/src/lib.rs

```diff
@@ -1,7 +1,6 @@
 #![recursion_limit = "1024"]
+#![allow(clippy::result_large_err)] // add this until refactor of `crate::error::ApcError result too large`
```

Comment:
I think it usually just wants us to `Box` probably just the `Backtrace`

Reply:
Tried that prior and clippy complained, loos like 1.67 might have fixed that?